lib/platform/github/index.ts

```diff
@@ -820,7 +820,12 @@ export async function findIssue(title: string) {
   };
 }
 
-export async function ensureIssue(title: string, body: string, once = false) {
+export async function ensureIssue(
+  title: string,
+  body: string,
+  once = false,
+  recreate = false
```

Comment:
Maybe we should call this "reopen" instead of "recreate"? And default to `true` for `reopen` to match existing behaviour.